packages/common/src/store/ui/modals/types.ts

```diff
@@ -151,6 +153,12 @@ export type StatefulModalsState = {
   DownloadTrackArchive: DownloadTrackArchiveModalState
   BuySellModal: BuySellModalState
   HostRemixContest: HostRemixContestModalState
+  ReceiveTokensModal: ReceiveTokensModalState
+}
+
+export type ReceiveTokensModalState = BaseModalState & {
+  tokenInfo?: TokenInfo
+  balance?: string
```

Comment:
ik it's in the design, but it feels weird showing a balance in the modal to me - at least when the balance is 0.